content/soarian/r4.md

```diff
@@ -0,0 +1,1099 @@
+---
+title: R4
+layout: api
+---
+
+# R4 Overview
+
+* TOC
+{:toc}
+
+Cerner’s Soarian Clinicals<sup>®</sup> implementation currently supports both the R4 First Normative Content (4.0.1) version and DSTU 2 Final (1.0.2) version of the Health Level Seven (HL7)<sup>®</sup> International Fast Healthcare Interoperability Resources (FHIR)<sup>®</sup> standard. Cerner’s implementation of the R4 version is ongoing and new resources and actions will be added over time.
+
+Existing DSTU 2 patient-facing applications will eventually need to be migrated to our R4 implementation. For this reason, Cerner strongly encourages development against R4 whenever possible.
+
+
+## Schema
```

Comment:
are you sure you want to put the whole capability statement here? It's only going to get bigger

Reply:
Clip the contents for better readability

Reply:
I discussed with Sudhir and reduced the  capability statement content